app/views/hyku_mailer/depositor_email.html.erb

```diff
@@ -1,34 +1,34 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <style>
-    body { font-family: Arial, sans-serif; line-height: 1.6; }
-    .email-container { max-width: 600px; margin: auto; padding: 20px; border: 1px solid #ccc; }
-    .footer { font-size: 12px; color: #666; margin-top: 20px; }
-    .highlight { color: #0056b3; }
-    .button {
-      display: inline-block;
-      padding: 10px 20px;
-      margin: 10px 0;
-      background-color: #0056b3;
-      color: white;
-      text-decoration: none;
-      border-radius: 5px;
-    }
-  </style>
-</head>
-<body>
-  <div class="email-container">
-    <p>Dear Author,</p>
-    <p>You had <%= pluralize(@statistics[:new_file_downloads], 'download') %> in <%= Date.today.prev_month.strftime("%B %Y") %> across your <%= pluralize(@statistics[:new_work_views], 'work') %> in <%= @application_name %>. Your current readership:</p>
-    <ul>
-      <li><%= pluralize(@statistics[:total_file_downloads], 'Total File Download') %></li>
-      <li><%= pluralize(@statistics[:total_work_views], 'Total Work View') %></li>
-    </ul>
-    <a href="<%= @url %>" class="button">VISIT MY DASHBOARD</a>
-    <div class="footer">
-      <p>These monthly reports are provided to you on behalf of <%= @application_name %>. For questions, comments, or to add more content and increase your readership and visibility as an author, please contact your repository administrator.</p>
+  <head>
+    <style>
+      body { font-family: Arial, sans-serif; line-height: 1.6; }
+      .email-container { max-width: 600px; margin: auto; padding: 20px; border: 1px solid #ccc; }
+      .footer { font-size: 12px; color: #666; margin-top: 20px; }
+      .highlight { color: #0056b3; }
+      .button {
+        display: inline-block;
+        padding: 10px 20px;
+        margin: 10px 0;
+        background-color: #0056b3;
+        color: white;
+        text-decoration: none;
+        border-radius: 5px;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="email-container">
+      <p>Dear Author,</p>
+      <p>You had <%= pluralize(@statistics[:new_file_downloads], 'new download') %> in <%= Date.today.prev_month.strftime("%B %Y") %> across your <%= pluralize(@statistics[:new_work_views], 'new work') %> in <%= @application_name %>. Your current readership:</p>
```

Comment:
all that I changed here was the addition of the word "new". I'm not sure why it looks like there are so many changes.

Reply:
and actually, this message doesn't really make sense. 

<img width="1093" alt="image" src="https://github.com/user-attachments/assets/439b00df-5b0b-45d4-a484-9182fe7165d4">

doesn't this imply there's been a new file download from one of the 6 new works, which i don't think is how these measurements work. instead of 6 new works should it be the total work count? or should language change to say there's been 6 new work views?

Reply:
I would think it should say something like "You had 1 new download and 6 new works". Assuming that's what was tracked.

Reply:
changed to 
![image](https://github.com/user-attachments/assets/717131a7-5119-4c54-9a01-7c96dd150219)